jenkins/Dockerfile-blossom.multi

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+###
+#
+# Dockerfile for multi-platform build
+#
+# Arguments Example:
+#       CUDA_VER=11.8.0
+#       UBUNTU_VER=20.04
+#       UCX_VER=1.15.0-rc6
+###
+
+ARG CUDA_VER=11.8.0
+ARG UBUNTU_VER=20.04
+ARG UCX_VER=1.15.0-rc6
+# multi-platform build with: docker buildx build --platform linux/arm64,linux/amd64 <ARGS> on either amd64 or arm64 host
+# check available official arm-based docker images at https://hub.docker.com/r/nvidia/cuda/tags (OS/ARCH)
+FROM --platform=$TARGETPLATFORM nvidia/cuda:${CUDA_VER}-runtime-ubuntu${UBUNTU_VER}
+ARG CUDA_VER
+ARG UBUNTU_VER
+ARG UCX_VER
+ARG ARCH=arm64
```

Comment:
What may happen if TARGETPLATFORM is `linux/amd64` but ARCH is arm64 ? I mean TARGETPLATFORM is not compatible with ARCH and UCX_ARCH.

Reply:
the docker build part would fail in next step `RUN update-java-alternatives --set /usr/lib/jvm/java-1.8.0-openjdk-${ARCH}`
```
# set amd64 platform but mismatched ARCH for JVM
```